packages/amplify-gen2-codegen/package.json

```diff
@@ -45,6 +45,7 @@
     "@aws-sdk/client-lambda": "^3.651.1",
     "@aws-sdk/client-s3": "^3.651.1",
     "aws-cdk-lib": "^2.127.0",
+    "ci-info": "^3.8.0",
```

Comment:
We don't need this dependency now right ?

Reply:
LGTM overall. Just had one minor comment about removing `ci-info` build time dependency since we moved it to runtime.